src/Main.java

```diff
@@ -1,76 +1,66 @@
-        } else if(agePeople == 7 || agePeople <= 17) {
-            System.out.println("Если человеку " + agePeople + ", то ему нужно ходить в школу");
-        } else if (agePeople >= 18 || agePeople <= 24) {
-            System.out.println("Если человеку " + agePeople + ", то ему нужно ходить в университет");
-        } else if (agePeople > 24) {
-            System.out.println("Если человеку " + agePeople + ", то ему пора ходить на работу");
+        System.out.println("Задача 4");
+        for (int i = 10; i > -11; i--) {
+            System.out.println(i);
         }
-        //задача 5
-        int kid=17;
-        if(kid<5) {
-            System.out.println("Если возраст ребёнка равен "+kid+" нельзя кататься на атракционе");
-        } else if(kid>5 && kid<14) {
-            System.out.println("Если возраст ребёнка равен "+kid+", то можно кататься в сопровождении взрослого");
-        } else if(kid>14){
-            System.out.println("Если возраст ребёнка равен "+kid+", то можно кататься без сопровождения взрослого");
+        //Задача 5
+        System.out.println("Задача 5");
+        for (int i = 1904; i < 2096; i = i+4) {
+            System.out.println(i + " год является високосным");
         }
         //Задача 6
-        int peoples=80;
-        int train=102;
-        int seatTrain=60;
-        int stayTrain=train-seatTrain;
-        if (peoples>=seatTrain && peoples<=seatTrain) {
-            System.out.println("Если в вагон хотят зайти "+peoples+" людей, то они могут занять сидячее место");
-        } else if(peoples>=stayTrain || peoples<=stayTrain){
-            System.out.println("Если в вагон хотят зайти "+peoples+" людей, то они могут занять сидячие и стоячие места");
-        } else if (peoples>train) {
-            System.out.println("Если в вагон хотят зайти " + peoples + " людей, то не получится, так как вагон будет забит");
+        System.out.println("Задача 6");
+        for (int i = 7; i < 105; i = i+7) {
+            System.out.println(i);
         }
         //Задача 7
-        int one=1;
-        int two=2;
-        int three=3;
-        if (one>two && one>three) {
-            System.out.println("Самое большое число это "+one);
-        } else if (two>three) {
-            System.out.println("Самое большое число это "+two);
-        } else if (three>two && three>one) {
-            System.out.println("Самое большое число это "+three);
+        System.out.println("Задача 7");
+        for (int i = 1; i < 1016; i = i*2) {
+            System.out.println(i);
+        }
+        //Задача 8
+        System.out.println("Задача 8");
+        int salary = 29000;
+        int total = 0;
+        for (int i = 0; i < 13; i++) {
+            total=total+salary;
+            System.out.println("Месяц " +i+ ", сумма накоплений равна " +total+" рублей");
+        }
+        //Задача 9
+        System.out.println("Задача 9");
+        int oneSalary = 29000;
+        int oneTotal = 0;
+        for (int i = 0; i < 13; i++) {
```

Comment:
```suggestion
        for (int i = 0; i < 12; i++) {
```
i лучше назвать month, также у тебя в году 13 месяцев, а надо 12